repose-aggregator/components/filters/header-normalization-filter/src/test/scala/org/openrepose/filters/headernormalization/HeaderNormalizationFilterTest.scala

```diff
@@ -48,23 +50,29 @@ class HeaderNormalizationFilterTest extends FunSpec with BeforeAndAfterEach with
   var filter: HeaderNormalizationFilter = _
   var servletRequest: MockHttpServletRequest = _
   var servletResponse: MockHttpServletResponse = _
-  var filterChain: FilterChain = _
-  var filterConfig: FilterConfig = _
-  var metricsService: MetricsService = _
-  var metricsMeters: MeterByCategorySum = _
+  private val filterChain = mock[FilterChain]
```

Comment:
Why make this change? It seems to me you made this test less safe, now if someone adds another mock they need to be sure manually reset it, with the other way it didn't matter.